docs/internal-documentation/distrib-testing/Ubuntu-Node.js-build-environment.md

```diff
@@ -16,14 +16,15 @@ been tested on Ubuntu 18.04, which we use here.
 
 3. Install packagages needed
    ```sh
-   sudo apt-get install build-essential
+   # Image manipulation librairies are indirect dependencies of the e2e tests modules
+   sudo apt-get install build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
    ```
```

Comment:
It is better to put the comment outside the shell box. It makes it easier to copy and paste.
````   
# Image manipulation librairies are indirect dependencies of the e2e tests modules
```sh
sudo apt-get install build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
```
````

Reply:
done